base/changes.txt

```diff
@@ -6,6 +6,14 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2024-10-27  Marcel Krüger  <Marcel.Krueger@latex-project.org>
+	* lttagging.dtx
+	Adapt two argument tagging sockets to preserve second argument when the socket is disabled.
```

Comment:
Change tagging sockets with two arguments todrop the first argument but return the second argument when the socket is disabled with \SuspendTagging.